src/assets/htmx.md

```diff
@@ -0,0 +1,347 @@
+---
+title: htmx
+authorName: Niraj Acharya
+authorAvatar: https://avatars.githubusercontent.com/u/122071597?v=4
+authorLink: https://github.com/nirajacharya2
+createdAt: Sep 29, 2023
+tags: js, html, go, htmx
+banner: https://raw.githubusercontent.com/JankariTech/blog/master/src/imgs/fallback_banner.png
+---
+
+Htmx is ui library written in JavaScript that extends the features of html. It can be extended to make AJAX requests, do CSS transitions, connect to WebSockets and handle Server Sent Events inside a html document with just some delacritive html attributes.
```

Comment:
```suggestion
Htmx is an UI library written in JavaScript that extends the features of html. It can be extended to make AJAX requests, do CSS transitions, connect to WebSockets and handle Server Sent Events inside a html document with just some declarative html attributes.
```
for "delacritive", I'm not sure if my suggestion is what you wanted to say...